content/docs/state-and-lifecycle.md

```diff
@@ -70,7 +70,8 @@ ReactDOM.render(
 
 To implement this, we need to add "state" to the `Clock` component.
 
-State is similar to props, but it is private and fully controlled by the component.
+State is similar to props, but it is private and fully controlled by the component. It is not necessary to convert the functional component to a ES6 class in order to set and access state. State Hooks can be used to use state without writing a class component.
```

Comment:
```suggestion
State is similar to props, but it is private and fully controlled by the component. It is not necessary to convert the function component to a ES6 class in order to set and access state. State Hooks can be used to use state without writing a class component.
```

See https://reactjs.org/docs/components-and-props.html#function-and-class-components